renting/setting-up-renterd/windows.md

```diff
@@ -47,156 +45,75 @@ To ensure you will not run into any issues with running `renterd` it is recommen
+Remove-Item -LiteralPath "$HOME\Downloads\renterd_windows_amd64.zip" -Recurse
 ```
-{% endtab %}
-{% endtabs %}
 
 {% hint style="warning" %}
 When you paste multi-line commands into PowerShell, you will be prompted with a warning. Make sure you have copied the entire command and click `Paste anyway` to proceed.
 {% endhint %}
 
-![Click `Paste anyway` to proceed with installation.](../../.gitbook/assets/renterd-install-screenshots/windows/01-renterd-multiline-warn.png)
-
-![Installation of renterd completed successfully.](../../.gitbook/assets/renterd-install-screenshots/windows/02-renterd-download-and-install.png)
-
----
+![Click `Paste anyway` to proceed with installation.](../../.gitbook/assets/renterd-screenshots/install/windows/02-renterd-windows-multiline-warn.png)
 
-## Creating a wallet
+![Installation of renterd completed successfully.](../../.gitbook/assets/renterd-screenshots/install/windows/03-renterd-windows-download-and-install.png)
 
-`renterd` uses BIP-39 12-word recovery phrases. To generate a new wallet recovery phrase, run the following command:
+## Verify `renterd` was installed successfully
 
-{% tabs %}
-{% tab title="Mainnet" %}
-```powershell
-cd $HOME\sia\renterd\; `
-.\renterd.exe seed
-```
-{% endtab %}
+Run the following command to see the version of `renterd` that was installed:
 
-{% tab title="Zen Testnet" %}
 ```powershell
-cd $HOME\sia\renterd_zen\; `
-.\renterd.exe seed
+cd $HOME\sia\renterd; `
+.\renterd.exe version
 ```
-{% endtab %}
-{% endtabs %}
-
-{% hint style="warning" %}
-A new 12-word recovery phrase will be generated. Make sure to store it in a safe place, as you will need this phrase to recover your wallet.
-{% endhint %}
-
-![](../../.gitbook/assets/renterd-install-screenshots/windows/03-renterd-seed.png)
-
----
 
-## Configure your `renterd.yml` file
+![](../../.gitbook/assets/renterd-screenshots/install/windows/04-renterd-windows-version.png)
 
-Under `$HOME\sia\renterd\bin` create a new text document named `renterd.yml`.
+## Configure `renterd`
 
-{% tabs %}
-{% tab title="Mainnet" %}
-```powershell
-New-Item -Path "$HOME\sia\renterd" -Name "renterd.yml" -ItemType "file"; `
-Start-Process "C:\WINDOWS\system32\notepad.exe" "$HOME\sia\renterd\renterd.yml"
-```
-{% endtab %}
+After installing `renterd`, you will need to configure it with a wallet seed and a password to unlock the web interface. There is an interactive configuration process that you can start by running the following command.
 
-{% tab title="Zen Testnet" %}
 ```powershell
-New-Item -Path "$HOME\sia\renterd_zen" -Name "renterd.yml" -ItemType "file"; `
-Start-Process "C:\WINDOWS\system32\notepad.exe" "$HOME\sia\renterd_zen\renterd.yml"
-```
-{% endtab %}
-{% endtabs %}
-
-Once Notepad loads, enter the following and configure it as needed.
-
-```yaml
-seed: your seed phrase goes here
-http:
-  password: your_api_password
-autopilot:
-  heartbeat: 5m
+cd $HOME\sia\renterd; `
```

Comment:
same here

edit: I guess for this one it's a little different because we might default the data directory to the current directory but that's only if you're on some very exotic OS or distro